src/Item_Ticket.php

```diff
@@ -205,7 +205,8 @@ public static function itemAddForm(Ticket $ticket, $options = [])
             return false;
         }
 
-        $canedit = ($ticket->can($params['id'], UPDATE)
+        $can_add_items = $_SESSION["glpiactiveprofile"]["helpdesk_hardware"] & pow(2, Ticket::HELPDESK_MY_HARDWARE) || $_SESSION["glpiactiveprofile"]["helpdesk_hardware"] & pow(2, Ticket::HELPDESK_ALL_HARDWARE);
```

Comment:
```suggestion
        $can_add_items = Session::haveRightsOr('helpdesk_hardware', [Ticket::HELPDESK_MY_HARDWARE, Ticket::HELPDESK_ALL_HARDWARE]);
```

no ?

Reply:
helpdesk_hardware isn't a regular right and this code was used other places so I thought it was required. I tested with no options selected for this right and then all options. These two cases worked. I then chose to see only All items and nothing was showing in the ticket with this proposed change. With the original fix, the one dropdown was showing as expected.